Cargo.toml

```diff
@@ -338,6 +338,7 @@ members = [
 	"substrate/frame/core-fellowship",
 	"substrate/frame/delegated-staking",
 	"substrate/frame/democracy",
+	"substrate/frame/distribution",
```

Comment:
I think we need to add the `pallet-opf` in this list as well. 

Because, the CI is not shouting at `pallet-distribution` which is mentioned earlier.

Reply:
It worked!! now we need to fix labels. Thanks!!!!

Reply:
Nice!
Leave the labels, only maintainers can add that and they'll do it.